scripts/upgrade

```diff
@@ -1,107 +1,156 @@
 #!/bin/bash
-set -eu
 
-source .fonctions       # Charge les fonctions génériques habituellement utilisées dans le script
+#=================================================
+# GENERIC START
+#=================================================
+# IMPORT GENERIC HELPERS
+#=================================================
 
-# Récupère les infos de l'application.
-app=$YNH_APP_INSTANCE_NAME
-
-# Source app helpers
+# source _common.sh
 source /usr/share/yunohost/helpers
 
+#=================================================
+# LOAD SETTINGS
+#=================================================
+
+app=$YNH_APP_INSTANCE_NAME
+
 domain=$(ynh_app_setting_get $app domain)
-path=$(ynh_app_setting_get $app path)
+path_url=$(ynh_app_setting_get $app path)
 admin=$(ynh_app_setting_get $app admin)
 final_path=$(ynh_app_setting_get $app final_path)
-db_pwd=$(ynh_app_setting_get $app mysqlpwd)
-db_user=$(ynh_app_setting_get $app db_user)
+db_name=$(ynh_app_setting_get $app db_name)
 
-CHECK_PATH      # Vérifie et corrige la syntaxe du path.
+#=================================================
+# ENSURE DOWNWARD COMPATIBILITY
+#=================================================
 
-# We make the app retro-compatible with previous versions
-if [[ -z "$final_path" ]];
-then
-   final_path="/var/www/$app"
-   ynh_app_setting_set "$app" final_path "$final_path"
-else
-   echo "${final_path} exists, we don't create it."
+# If db_name doesn't exist, create it
+if [ -z $db_name ]; then
+	# In older version, db_name was always phpmyadmin
+	db_name=phpmyadmin
+	ynh_app_setting_set $app db_name $db_name
 fi
 
-# In older version, db_user was always phpmyadmin
-if [[ -z "$db_user" ]];
-then
-   db_user="phpmyadmin"
-   ynh_app_setting_set "$app" db_user "$db_user"
-else
-   echo "${db_user} exists, we don't create it."
+# If final_path doesn't exist, create it
+if [ -z $final_path ]; then
+	final_path=$(ynh_sanitize_dbid $app)
```

Comment:
I guess here you want `final_path="/var/www/$app"` instead...

Reply:
Yes my bad!

Reply:
Fixed by https://github.com/YunoHost-Apps/phpmyadmin_ynh/pull/63/commits/47b1bc11fb54f3c1e6b117d22ce7b5366b71fc3e